stonesoup/plotter.py

```diff
@@ -271,6 +275,32 @@ def plot_tracks(self, tracks, mapping, uncertainty=False, particle=False, track_
         else:
             self.ax.legend(handles=self.handles_list, labels=self.labels_list)
 
+    def plot_density(self, state_sequences: Sequence[StateMutableSequence], index=-1, mapping=(0, 2), n_bins=300,
+                     **kwargs):
+
+        if index is None:
```

Comment:
Don't need this check with default value of -1. Unless you are expecting users to manually input 'index=-1'.

Reply:
This should have been documented more clearly. If you manually set `index=None` the code will plot all the states in the sequences